docs/migration-guides/classic-iqp-to-cloud-iqp.mdx

```diff
@@ -44,7 +48,13 @@ The Lite Plan is deprecated and will be removed on 14 March 2025.
 
 ## Current IBM Quantum Platform users
 
-Instead of accessing [IBM Quantum Platform Classic](https://quantum.ibm.com/), you will now access the [updated version of IBM Quantum Platform](https://quantum.cloud.ibm.com/), which is integrated with IBM Cloud.  Because of this integration, all users need to follow the onboarding instructions before using the new platform.
+Instead of accessing [IBM Quantum Platform Classic](https://quantum.ibm.com/), you will now access the [upgraded version of IBM Quantum Platform](https://quantum.cloud.ibm.com/), which is integrated with IBM Cloud.  Because of this integration, all users need to follow the onboarding instructions before using the new platform.
+
+### Save results from IBM Quantum Platform Classic
+
```

Comment:
For this "important" admonition about saving jobs, I would update copy to something more like:

"Jobs that were run on IBM Quantum Platform Classic will not be available after that version of platform is sunset on July 1, 2025. Open and Pay-as-you-go users should therefore save any workloads data from the classic platform prior to that date by following these steps. Premium, Standard, and Dedicated service plan users should wait to receive guidance from their administration on how and if to download historical data."

Reply:
pay-go = standard, so those users don't have to download their content; their jobs will still be available.

Reply:
> pay-go = standard, so those users don't have to download their content; their jobs will still be available.

OMG yes you are right! let's just call out Open users in this line then.


> Fran told me that premium & dedicated users will have their data migrated for them.
Yes, that is correct, which is why I was just trying to allude to the fact that they don't need to go through these steps, and should wait to learn more from their admins.